client/blocks/reader-combined-card/index.jsx

```diff
@@ -137,4 +139,23 @@ class ReaderCombinedCard extends React.Component {
 	}
 }
 
-export default localize( ReaderCombinedCard );
+function combinedCardPostKeyToKeys( postKey ) {
+	if ( ! postKey || ! postKey.postIds ) {
+		return [];
+	}
+
+	const feedId = postKey.feedId;
+	const blogId = postKey.blogId;
+	return postKey.postIds.map( postId => ( { feedId, blogId, postId } ) );
+}
+
+const getPostsByKeys = ( state, postKeys ) =>
```

Comment:
note: this is inefficient. Ideally we can change the data structures involved here or make a memoized selector so that it doesn't cause needless rerenders